### ISSUE TITLE
The last candle vs another one in specific...

### ISSUE THREAD

**Issue by unzer** [Fruitfella/ROBOT-binary#17]
Hi @Fruitfella 
I tried, but failed... how is it possible do that:
>IF the last candle (can be the open, close, maximum...) is greater then the before, purchase rise....
![fig1](https://cloud.githubusercontent.com/assets/26580460/26477626/6110b854-419d-11e7-9713-5357aee99b44.jpg)
![fig2](https://cloud.githubusercontent.com/assets/26580460/26477627/61145c48-419d-11e7-9a97-8b3d90cbec80.jpg)

**Comment by Fruitfella** [comment]
please elaborate more about ur system. i see many scenario from the pict, idk which one is ur system.

**Comment by Fruitfella** [comment]
@unzer please elaborate more about ur system. i see many scenario from the pict, idk which one is ur system.

**Comment by Fruitfella** [comment]
![patern](https://cloud.githubusercontent.com/assets/26277327/26516382/bddbca98-42af-11e7-911a-288a093b4b6d.png)
So, which one is ur strategy @unzer ?

**Comment by unzer** [comment]
Hi @Fruitfella , sorry for my delay...
You are correct, there are more than one strategy, Is it possible show me one? You choose!

**Comment by Fruitfella** [comment]
the ill take strategy number 3.
how about ur 2nd picture? it seems the candle1 is opposite of candle4
do i need to combine the strategy from picture 1 and picture 2?

**Comment by unzer** [comment]
@Fruitfella
Yes how can I develop the strategy number 3? The candle 1 closed > candle4...

**Comment by Fruitfella** [comment]
@unzer no. number 3 isnt candle1>candle4 but **candle1 body > candle2 body** , see my pict above.
as for candle1 and candle4, it isnt about which one is **bigger** , but they are on **opposite direction** . candle1 is bullish (green) while candle4 is bearish (red).
im testing my bot now, so i cant give u a sample right away.
wait until i finish it. well it would take a whole day, _lol_

**Comment by unzer** [comment]
Hi @Fruitfella 
I got it!!!
![image](https://cloud.githubusercontent.com/assets/26580460/26522024/9003dbd8-42cd-11e7-93f6-b12c1318cc6f.png)

**Comment by Fruitfella** [comment]
no @unzer , thats wrong.
this is the bot made according to the picture
![untitled1](https://cloud.githubusercontent.com/assets/26277327/26522404/1130e1a4-432a-11e7-9168-0e2774d07621.png)

![unzerbot](https://cloud.githubusercontent.com/assets/26277327/26522401/065cc248-432a-11e7-9cd2-bcd15345bda7.PNG)

**Comment by Fruitfella** [comment]
This is after I combine the strategies from ur 1st picture and 2nd picture @unzer .
![untitled1](https://cloud.githubusercontent.com/assets/26277327/26522475/136b2ea0-432c-11e7-8d0b-7b24c1df74e6.png)
![unzerbot2](https://cloud.githubusercontent.com/assets/26277327/26522479/1cf06c56-432c-11e7-8bc6-6208da2a9073.PNG)

**Comment by Fruitfella** [comment]
@unzer 
I posted the bot <a href="https://gist.github.com/Fruitfella/d7b2cf19e4705e17bcb6d60c43d9188f">here</a>!

**Comment by nobleb2007** [comment]
Thanks for the digit differ bot u help me with. Im still testing it. God bless you @nobleb2007. Fruitfella im grateful

**Comment by nobleb2007** [comment]
I need someone that can put me through on how to add Martindale to the bot. I have idea but can implement it without martinglae. PLEASE

**Comment by Fruitfella** [comment]
Hi @nobleb2007 
if u have any question that isnt relevant to any existing issues, please make new issue, so i wont be confuse.
to create new issue, click the green button `New issue` on top-right corner.
